### ISSUE TITLE
[L] Lobby Chat command

### ISSUE THREAD

**Issue by eykamp** [bitfighter/bitfighter#550]
Add a lobby chat to the editor, similar to [G]lobal chat in-game

**Comment by raptor** [comment]
F5 works in the editor - is this something different?

**Comment by eykamp** [comment]
Yes -- this would be more like the in-game chat interface... you'd see 
messages appear on the screen, and reply with a similar UI. Unclear 
description!

**Comment by eykamp** [comment]
The idea is to make it easier to see chat messages and get involved in 
chatting while editing.

**Comment by Xelor41023** [comment]
Still unclear so other player would be able to see what you are editing and suggest ideas and chat with you about the design?

**Comment by eykamp** [comment]
No, this case is only to make lobby chat more accessible in the editor.
